src/storage.rs

```diff
@@ -22,25 +21,42 @@ pub trait HasIndex {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
-pub enum TypeStorer {
+pub enum IndexedTypeStorer {
     Redact(redact::RedactStorer),
     Mongo(mongodb::MongoStorer),
+    Mock(tests::MockIndexedStorer),
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum TypeStorer {
     GoogleCloud(gcs::GoogleCloudStorer),
-    Mock(tests::MockStorer),
+    Mock(tests::MockIndexedStorer),
```

Comment:
Should this a non-indexed mock?

Reply:
fixed by adding TypeStorer enum above IndexedTypeStorer and NonIndexedTypeStorer